lib/recording/RecordingEntrySession.js

```diff
@@ -47,7 +49,8 @@ class RecordingEntrySession {
         let now = new Date();
         let entryId = entryObj.entryId;
         let recordedEntryId = entryObj.recordedEntryId;
-        if (entryObj.recordStatus === kalturaTypes.KalturaRecordStatus.APPENDED){
+        this.entryServerNodeId = entryServerNodeId;
```

Comment:
no. entryObj doesn't include entryServerNodeId. Noam updated BE code so it won't happen.